cluster/gce/container-linux/configure-helper.sh

```diff
@@ -1358,8 +1358,12 @@ function start-lb-controller {
     echo "Start GCE L7 pod"
     prepare-log-file /var/log/glbc.log
     setup-addon-manifests "addons" "cluster-loadbalancing/glbc"
-    cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/glbc.manifest" \
-       /etc/kubernetes/manifests/
+
+    local -r glbc_manifest="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/glbc.manifest"
+    if [[ ! -z "${GCE_OVERRIDE_GLBC_IMAGE:-}" ]]; then
+      sed -i "s@image:.*@image: ${GCE_OVERRIDE_GLBC_IMAGE}@" ${glbc_manifest}
```

Comment:
`s/image:.*$/`? 

Also, isn't it better to replace the manifest entirely, rather than replacing just the image?

quote `"${glbc_manifest}"`

why not use the '/' character? It doesn't appear in your string...

Reply:
Because image is a full URL which includes "/". In the same file, all sed are using @ as deliminator.

Just the image are overrided. No need to change other stuff, right?